package.json

```diff
@@ -70,13 +70,13 @@
     "ml-array-xy-equally-spaced": "^1.2.1",
     "ml-baseline-correction-regression": "^1.0.0",
     "ml-conrec": "^3.2.1",
-    "ml-gsd": "^12.0.0",
+    "ml-gsd": "12.1.0",
```

Comment:
@jobo322 Can you explain how you update dependencies? I don't understand why the caret is removed so often in your updates.

Reply:
I use `npm i [packageName]@[version]`, I don't use this caret in this case because I was not sure about the ml-gsd version, it was intentional and I did returns the caret symbol.

Reply:
I can not reproduce this. The caret is not removed when I do `npm i ml-gsd@12.1.0` For you Alejandro it is being removed ? 

There is an option in npm that allows to force the version: https://stackoverflow.com/questions/30656189/make-npm-install-save-add-a-strict-version-to-package-json

Could you do `npm config ls -l` and check that `save-exact = false`

Reply:
yes, exact version flag is false, the use of --save-exact is the best way as Michael let me know to install a pre-release

Reply:
So did you do `npm i --save-exact ml-gsd@latest` ?

How did you do to remove this `^`

Reply:
No, this is the current way to install a fixed version. I remove the caret manually for ml-gsd. I did it because I was confused, as to why I got installed a different version, for nmr-processing I got installed always version 12.0.0, so in order to test the current state of the code I used a fixed version of ml-gsd to be sure.

I used this fixed version as we use the fixed version of pre-release. Now I understand that npm prefers the stable version instead of a pre-release version, it was the source of my confusion.

I catch the idea that we always use the caret from the above comments, I will only use the fixed version for pre-release versions